react/nwb.config.js

```diff
@@ -11,5 +13,34 @@ module.exports = {
   },
   babel: {
     stage: 1
+  },
+  webpack: {
```

Comment:
Since we know this will only work with the umd published version of Mayflower React, what are the implications at this point for consuming this in search?